contentpacks/khanacademy.py

```diff
@@ -79,8 +79,34 @@ def retrieve_language_resources(lang: str, version: str) -> LangpackResources:
     return LangpackResources(topic_data, content_data, exercise_data, subtitle_data, kalite_catalog, ka_catalog, dubbed_video_mapping)
 
 
-def retrieve_subtitles(videos: list, lang="en") -> dict:
-    return {}
+def retrieve_subtitles(videos: list, lang="en") -> list:
+    #videos => contains list of youtube ids
+    #return list of youtubeids that were downloaded
+    downloaded_videos = []
+    not_downloaded_videos = []
+    for youtube_id in videos:
+        request_url = "https://www.amara.org/api2/partners/videos/?format=json&video_url=http://www.youtube.com/watch?v=%s" % (
+        youtube_id)
+        response =  requests.get(request_url)
```

Comment:
are you referring for line 90 or 100? 90 will always return a 200 regardless of a bad youtube id, unless the server is down then I don't see a point using it over there, unless you meant 100